x/accesscontrol/keeper/keeper.go

```diff
@@ -125,39 +126,46 @@ func (k Keeper) SetDependencyMappingDynamicFlag(ctx sdk.Context, messageKey type
 
 type ContractReferenceLookupMap map[string]struct{}
 
-func (k Keeper) GetWasmDependencyMapping(ctx sdk.Context, contractAddress sdk.AccAddress, senderBech string, msgBody []byte, applySelector bool, circularDepLookup ContractReferenceLookupMap) (acltypes.WasmDependencyMapping, error) {
+func (k Keeper) GetRawWasmDependencyMapping(ctx sdk.Context, contractAddress sdk.AccAddress) (*acltypes.WasmDependencyMapping, error) {
+	store := ctx.KVStore(k.storeKey)
+	b := store.Get(types.GetWasmContractAddressKey(contractAddress))
+	if b == nil {
+		return nil, sdkerrors.ErrKeyNotFound
+	}
+	dependencyMapping := acltypes.WasmDependencyMapping{}
+	if err := k.cdc.Unmarshal(b, &dependencyMapping); err != nil {
+		return nil, err
+	}
+	return &dependencyMapping, nil
+}
+
+func (k Keeper) GetWasmDependencyMapping(ctx sdk.Context, contractAddress sdk.AccAddress, senderBech string, msgBody []byte, circularDepLookup ContractReferenceLookupMap) (*types.AccessOperationSet, error) {
```

Comment:
we still would like to return a slice of access operations since that's what the dependency generator (in sei-chain repo) will pass back to the dependency graph generation logic.

(see comment below on having list conversion helper function)

Reply:
updated